src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocsummarylocation/InputJavadocSummaryLocationTrailingSpace.java

```diff
@@ -0,0 +1,32 @@
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocsummarylocation;
+
+/**
+ * config:  default
+ * location="SECOND_LINE"
+ */
+public final class InputJavadocSummaryLocationTrailingSpace {
+
+    /**  
+     * ^^ There is a trailing space in the first line. // OK
+     */
+    private int space;
+
+    /**********************
+     * Trailing asterisks in the first line * // OK
+     *
+     **********************/
+    private int asterisks;
+
+    /** ******
```

Comment:
lets add case:
```
    /** ******
      * There is a space, then leading asterisks that trailing spaces in the first line. // OK ? or ?
```

Reply:
Done. Now InputTrailingSpace has all cases from `space` to `space-asterisk-space-asterisk`.